src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  const { name, age, isMarried, partnerName, sex } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+
+      <p className="Person__partner">
+        {isMarried
+          ? `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
```

Comment:
move this to the variable
```suggestion
          ? `${partnerName} is my ${partnet}`
```